extras/controllerClient/readme.md

```diff
@@ -8,30 +8,43 @@ The client API is implemented as a dll (dynamic link library). The functions in
 
 ## What is Included?
 
-Note: The header and library files are built along with nvdaHelper. If you want to build them yourself, please see readme.txt in the NVDA source distribution for instructions. Alternatively, you can find here a [pre-built package](http://www.nvda-project.org/nvdaControllerClient/nvdaControllerClient_20100219.7z).
+**Note:**
 
-Versions of the libraries and headers are supplied for both x86 and x64 applications. They have been split in to x86 and x64 directories, respectivly.
+The header and library files are built along with nvdaHelper.
+If you want to build them yourself, please see `readme.txt` in the NVDA source distribution for instructions.
 
-Each directory contains the following files, where * denotes 32 for x86 and 64 for x64:
+Versions of the libraries and headers are supplied for both `x86` and `x64` applications. They have been split in to `x86` and `x64` directories, respectivly.
```

Comment:
For some time now we also compile Controller Client for ARM64. Shouldn't it be mentioned as a supported architecture here?